install

```diff
@@ -45,6 +48,36 @@ jb install
 # Install apps into separate namespaces
 if [[ "$MODE" == "app-only" ]]; then
     install_env tns-cloud       # Only install (modified) TNS demo app into existing K8s cluster 
+elif [[ "$MODE" == "grafana-cloud" ]]; then
+		read -p "Please enter your org slug: " ORGSLUG
+		read -p "Please enter your API-Key: " APIKEY
+		INSTANCESFILE=$(mktemp)
+		curl --silent -H "Authorization: Bearer $APIKEY" "https://grafana.com/api/orgs/$ORGSLUG/instances" > $INSTANCESFILE
+		STACKS=$(jq -r '[.items[].slug] | join(",")' $INSTANCESFILE)
+		read -p "Please select your stack (available: $STACKS): " STACK
+
+		read -r -d '' GCLOUD_CONFIG <<EOF
+{
+  _config+:: {
```

Comment:
probably not great to write an API key into a jsonnet file, but for just a demo i think it's fine.